Frends.Kafka.Produce/Frends.Kafka.Produce/Definitions/Result.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Frends.Kafka.Produce.Definitions;
+
+/// <summary>
+/// Task's result.
+/// </summary>
+public class Result
+{
+    /// <summary>
+    /// Message sent.
+    /// </summary>
+    /// <example>true</example>
+    public bool Success { get; private set; }
+
+    /// <summary>
+    /// Persistence status.
+    /// </summary>
+    public string Status { get; private set; }
+
+    /// <summary>
+    /// Message.
+    /// </summary>
+    public string Message { get; private set; }
+
+    /// <summary>
+    /// Timestamp.
```

Comment:
Which timestamp is this? When task is executed or is this reported back by Kafka call? Would be nice to clarify.